prowler/providers/aws/services/eventbridge/eventbridge_schema_registry_cross_account_access/eventbridge_schema_registry_cross_account_access.py

```diff
@@ -0,0 +1,70 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.eventbridge.schema_client import schema_client
+
+
+class eventbridge_schema_registry_cross_account_access(Check):
+    def execute(self):
+        findings = []
+        for registry in schema_client.registries.values():
+            report = Check_Report_AWS(self.metadata())
+            report.resource_id = registry.name
+            report.resource_arn = registry.arn
+            report.resource_tags = registry.tags
+            report.region = registry.region
+            report.status = "PASS"
+            report.status_extended = f"EventBridge schema registry {registry.name} does not allow cross-account access."
+            if registry.policy and "Statement" in registry.policy:
```

Comment:
Again, function for this please, you could re-use a lot of code.

Reply:
Done